CREDITS.txt

```diff
@@ -45,20 +45,20 @@ and minecraft clients.
 The code is authored by cpw.
 
 It began by partially implementing an API defined by the client side ModLoader, authored by Risugami.
-http://www.minecraftforum.net/topic/75440-
+https://www.minecraftforum.net/topic/75440-
 This support has been dropped as of Minecraft release 1.7, as Risugami no longer maintains ModLoader.
 
 It also contains suggestions and hints and generous helpings of code from LexManos, author of MinecraftForge.
-http://www.minecraftforge.net/
+https://minecraftforge.net/
 
-Additionally, it contains an implementation of topological sort based on that 
-published at http://keithschwarz.com/interesting/code/?dir=topological-sort
+Additionally, it contains an implementation of topological sort based on that
+published at https://keithschwarz.com/interesting/code/?dir=topological-sort
 
 It also contains code from the Maven project for performing versioned dependency
-resolution. http://maven.apache.org/
+resolution. https://maven.apache.org/
 
-It also contains a partial repackaging of the javaxdelta library from http://sourceforge.net/projects/javaxdelta/
-with credit to it's authors.
+It also contains a partial repackaging of the javaxdelta library from https://sourceforge.net/projects/javaxdelta/
+with credit to its authors.
 
 Forge Mod Loader downloads components from the Minecraft Coder Pack
 (http://mcp.ocean-labs.de/index.php/Main_Page) with kind permission from the MCP team.
```

Comment:
Is this still relevant?

Reply:
Erh, i don't believe so. It's all in house now.
Best leave the MCP credit in though, since their work was pivotal for Forge to be a thing

Reply:
Okay, I won't touch it.